src/Describer/InputDescriber/FormInputDescriber.php

```diff
@@ -42,12 +44,54 @@ public function describe(Input $input, Operation $operation, string $httpMethod)
         if ($input->isInQuery()) {
             $operation->parameters = array_merge($operation->parameters, $this->queryDescriber->describe($form));
         } elseif ($input->isInBody()) {
-            $operation->requestBody = new RequestBody(
-                [
-                    'required' => true,
-                    'content' => $this->bodyDescriber->describe($form, $httpMethod),
-                ]
-            );
+            $describedBody = $this->bodyDescriber->describe($form, $httpMethod);
+
+            if ($operation->requestBody === null) {
+                $operation->requestBody = new RequestBody(
+                    [
+                        'required' => true,
+                        'content' => $describedBody,
+                    ]
+                );
+            } else {
+                Assert::isInstanceOf($operation->requestBody, RequestBody::class);
```

Comment:
Is this check rally needed ?

Reply:
According to docblock `$operation->requestBody` might be also a `Reference` but we don't support this scenario currently.

Reply:
Could you add a comment for this ?